src/elements/content-sidebar/activity-feed/version/CollapsedVersion.js

```diff
@@ -83,9 +84,7 @@ const CollapsedVersion = (props: Props): React.Node => {
                         aria-label={intl.formatMessage(messages.getVersionInfo)}
                         className="bcs-Version-info"
                         data-resin-target={ACTIVITY_TARGETS.VERSION_CARD}
-                        onClick={() => {
-                            onInfo({ versions });
```

Comment:
What's passed into this cb is only used when `hasVersions` prop is `false` and `onVersionHistoryClick` prop is passed into BUIE. There doesn't seem to be a standardized format, since the CollapsedVersion and Version components pass back different data. Because UAA doesn't have the `versions` array like the v2 api, I decided to use the end version of the event and match what was passed in the `Version.js` component (`id` and `version_number`). This cb is not used in EUA.

Reply:
yup